src/main/java/roomescape/payment/service/usecase/PaymentQueryUseCase.java

```diff
@@ -0,0 +1,19 @@
+package roomescape.payment.service.usecase;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import roomescape.common.exception.NotFoundException;
+import roomescape.payment.domain.Payment;
+import roomescape.payment.repository.PaymentRepository;
+
+@Service
+@RequiredArgsConstructor
+public class PaymentQueryUseCase {
+
+    private final PaymentRepository paymentRepository;
+
+    public Payment get(final Long id) {
+        return paymentRepository.findById(id)
+                .orElseThrow(() -> new NotFoundException("id에 해당하는 결제 내역을 찾을 수 없습니다."));
```

Comment:
```suggestion
                .orElseThrow(() -> new NotFoundException("id: 1에 해당하는 결제 내역을 찾을 수 없습니다."));
```

이런 식으로 실제 id값을 넣어주면 예외 발생 시 처리를 좀 더 쉽게 할 수 있지 않을까요?

Reply:
동의해요! 사용자에게 보여지는 것이 아닌 내부적으로 남는 메시지이니, 추적에 용이하도록 ID까지 기입하는 게 더 좋겠네요.

반영해두었어요.